README.md

```diff
@@ -37,25 +37,38 @@ All clients supporting the Clique engine (EIP-225) are able to sync Görli.
 
 ##### Go-Ethereum
 
-You can connect Geth to Görli using the genesis provided in this repository:
+###### Setup Instructions
 
-Note on Bootnodes:
-You can connect to the bootnodes by copying `./geth/static-nodes.json` into the data directory (commonly .ethereum) and then run the steps below.
+1. First, install geth if you don’t already have it. Download it here: https://geth.ethereum.org/install/
```

Comment:
IMHO, since we are not officially launched yet, it's fine to assume users know how to install geth